drivers/planar/clarity_matrix.cr

```diff
@@ -0,0 +1,111 @@
+module Planar; end
+
+require "placeos-driver/interface/powerable"
+
+# Documentation: https://aca.im/driver_docs/Planar/020-1028-00%20RS232%20for%20Matrix.pdf
+#  also https://aca.im/driver_docs/Planar/020-0567-05_WallNet_guide.pdf
+
+class Planar::ClarityMatrix < PlaceOS::Driver
+  include Interface::Powerable
+
+  # Discovery Information
+  generic_name :VideoWall
+  descriptive_name "Planar Clarity Matrix Video Wall"
+
+  #   implements :device # don't need?
+
+  def on_load
+    # Communication settings
+    queue.wait = false
+    transport.tokenizer = Tokenizer.new("\r")
+  end
+
+  def on_unload
+  end
+
+  def on_update
+  end
+
+  def connected
+    schedule.every(60.seconds) { do_poll }
+    do_poll
+  end
+
+  def disconnected
+    # Disconnected may be called without calling connected
+    #   Hence the check if timer is nil here
+
+    schedule.clear
+  end
+
+  # def power?(options = {}, &block)
+  def power?(**options)
+    #  no optional block in crystal unless overloading
+    # options[:emit] = block if block_given?
+
+    # options[:wait] = true
+    # options[:name] = :pwr_query
+    send("op A1 display.power ? \r", **options)
+  end
+
+  # def power(state, broadcast_ip = false, options = {})
+  # what does broadcast_ip do here??
+  def power(state : Bool = false)
+    puts "power method running" 
+    # self[:power] = state
+    # send("op A1 display.power = off \r")
+    if state == true
+      send("op A1 display.power = on \r") # changed ** -> A1: "Power should only be sent to display A1"
+      power?
```

Comment:
It looks like the power state gets updated in remove to the power on command above, this second query can be removed so there's not unnecessary comms.